grid/chat/channels/maps/channels.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable jest/no-export */
+
+import { RecordingProcessOptions, SuperfaceTest } from '@superfaceai/testing';
+
+export const getChannelsTest = (
+  provider: string,
+  options?: { server?: string; recordingOptions?: RecordingProcessOptions }
```

Comment:
I did not find use-case to retrieve servers. Are we going to add another profile for it?

Reply:
I had to implement it for messaging demo, so I can open PR for it 🙌🏻